### PR TITLE
Update /careers/design page

### LINKED ISSUE #80: Careers - Update /careers/design based on the copydoc

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Update '/careers/design' page
- Add `meta_copydoc` to `/careers/all` and `/careers/design`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/design
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](https://docs.google.com/document/d/1zpAVnQ1qE-oAmZoxTpfOPFu12zbotBqrD_To9ry0oAQ/edit#)

## Issue / Card

Fixes #80

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -7,10 +7,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description"
       content="{% block meta_description %}Canonical produces Ubuntu, provides commercial services for Ubuntu’s users, and works with hardware manufacturers, software vendors and cloud partners to certify Ubuntu.{% endblock %}" />
+    <meta name="copydoc" content="{% block meta_copydoc %}https://drive.google.com/drive/u/1/folders/0B8feV0jqaac3TmExU3NDcHhFTGc{% endblock %}">
```
Comment (sowasred2012) [inline templates/base.html]: Not sure this link is correct:
- it shouldn't have `/u/1` in the URL, that's only there if you have multiple Google accounts (in this case '1' selects the second account you're logged in with, which may not be a user's Canonical account).
- https://drive.google.com/drive/folders/0B8feV0jqaac3TmExU3NDcHhFTGc links to a folder, is that expected?

Comment (solazio) [reply templates/base.html]: @sowasred2012 yes, that's default in case no link is provided for a specific page.

Comment (solazio) [reply templates/base.html]: @sowasred2012 ahh, I see what you say. will amend it.

File: templates/partial/_careers-available-roles.html [hunk 2]
```diff
@@ -36,11 +36,9 @@
       </div>
     </div>
   </div>
-  <div class="u-fixed-width p-notification u-no-margin--bottom js-filter__no-results u-hide">
-    <p class="p-notification__response">
-      Sorry, there are no open positions matching your criteria.
-    </p>
-  </div>
+  <h4 class="js-filter__no-results u-hide">
```
Comment (sowasred2012) [inline templates/partial/_careers-available-roles.html]: This header and the one on line 66 should both either be `h2`s or `p`s (with h4/h3 classes if need be), the heirarchy at the moment goes straight from `h1` to whichever one is displayed.

**Pull request merged**